src/index.html

```diff
@@ -3,11 +3,335 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>My BiKE</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      rel="icon"
+      type="images/x-icon"
+      href="images/favicon.png"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Landing Page</h1>
+    <header class="header" id="page-top">
+      <div class="container">
+        <div class="header__top short-navigation">
+          <a href="#page-top" class="short-navigation__logo">
+            <img src="images/logo-white.png"
```

Comment:
```suggestion
            <img 
              src="images/logo-white.png"
```
Consider fixing indentations according to the code style guide